homeassistant/components/ialarm/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "pin": "[%key:common::config_flow::data::pin%]"
```

Comment:
The `pin` string isn't in the user form schema.

Reply:
By the end of today I'll open a PR to fix them.